chartfx-chart/src/main/java/de/gsi/chart/viewer/DataViewWindow.java

```diff
@@ -413,6 +431,18 @@ public BooleanProperty detachedProperty() {
         return dragCursor;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
This is essentially needed that the same window/Pane/item is not added twice when switching from the detached to visibleChildren list. Added this, because sometimes the original DataViewWindow frame is being lost and regenerated when added back...